api/getProject.go

```diff
@@ -0,0 +1,86 @@
+package api
+
+import (
+	"eshaanagg/lfx/database/handlers"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+)
+
+func getProject(c *gin.Context) {
+	client := handlers.New()
+	defer client.Close()
+
+	orgID := c.Param("id")
+	projectID := c.Param("projectId")
+
+	org := client.GetOrganizationByID(orgID)
+	if org == nil {
+		c.IndentedJSON(http.StatusNotFound, gin.H{
+			"message": "There is no organization with this id",
+		})
+		return
+	}
+
+	// Using GetProjectByProjectId to get the project by project ID
+	projects, err := client.GetProjectByProjectId(projectID)
+	if err != nil {
+		fmt.Print(projectID, orgID)
+		c.IndentedJSON(http.StatusNotFound, gin.H{
+			"message": "Project not found",
+		})
+		return
+	}
```

Comment:
After this check, ensure that the `organizationId` of the fetched project matches the `id` of the organization supplied. If not, return a suitable error.

Reply:
Added a check with an error message and bad status.